BloomGameStudio/PuzzleService
# Puzzle verification

**Changes**

verification.go:

- Take a puzzle, and check a given solution to see if it is correct
- Mock solution for getCorrectSolutionForPuzzle


verification_test.go:

- Tests for successful & unsuccessful puzzle solution verification
- Tests for invalid inputs from the client

**How to test**

1. Open Insomnia or your client of choosing
2. Sent a POST request to http://localhost:3000/verify
3. Ensure your JSON body contains the ID of 1, and the "Correct!" string
